src/main/kotlin/com/jetpackduba/gitnuro/ui/dialogs/settings/SettingsDialog.kt

```diff
@@ -61,6 +61,7 @@ val settings = listOf(
     SettingsEntry.Entry(Res.drawable.layout, "Layout") { Layout(it) },
     SettingsEntry.Entry(Res.drawable.schedule, "Date/Time") { DateTime(it) },
     SettingsEntry.Section("GIT"),
+    SettingsEntry.Entry(Res.drawable.folder, "Environment") { Environment(it) },
```

Comment:
I'm not a big fan of having this under the "Git" section but I can change that later after merging